src/lib/server/cart.ts

```diff
@@ -142,6 +171,14 @@ export async function addToCartInDb(
 			params.customPrice.amount,
 			toCurrency(params.customPrice.currency, product.price.amount, product.price.currency)
 		);
+	} else if (variationPriceDelta > 0) {
```

Comment:
@Tirodem can you confirm that we don't allow a variation with negative pricing to decrease the price below the product's base price?

For example if there's a product at 100€ and a variation with -10€, if I chose that variation, how much do I pay?

This seems suspect to me

Reply:
We removed negative variation because, for a 1st version, it introduced complex needs.
So I confirm we aren't supposed to make negative prive variation, only null or positive.

Reply:
I have already changed the front input to take min 0

Reply:
it should be the backend input on the admin too

and this `variationDelta > 0` condition can be removed

Reply:
We have this in `productForm.svelte `
```
	Price difference<input
		                 type="number"
				 name="variations[{i}].price"
				class="form-input"
				value={variation.price}
				min="0"/>
```

Reply:
See comment here too: https://github.com/B2Bitcoin/beBOP/pull/1491#discussion_r1806157934